### PR TITLE
Better error messages for openmrs_start_datetime

### PR DESCRIPTION
**Pull request by kaapstorm**
Provide better feedback if the "openmrs_start_datetime" parameter doesn't return a valid value.

Feature flag: "Enable OpenMRS integration"

### CODE REVIEW

File: corehq/motech/openmrs/handler.py [hunk 1]
```diff
@@ -197,22 +198,31 @@ def run(self):
...
-                    stop_datetime = form_config.openmrs_start_datetime.serialize(
-                        cc_start_datetime + timedelta(days=1) - timedelta(seconds=1)
-                    )
+                    stop_datetime = form_config.openmrs_start_datetime.serialize(cc_stop_datetime)
                 else:
                     cc_start_datetime = string_to_utc_datetime(self.form_json['form']['meta']['timeEnd'])
+                    cc_stop_datetime = cc_start_datetime + timedelta(days=1) - timedelta(seconds=1)
                     start_datetime = to_omrs_datetime(cc_start_datetime)
-                    stop_datetime = to_omrs_datetime(cc_start_datetime + timedelta(days=1) - timedelta(seconds=1))
+                    stop_datetime = to_omrs_datetime(cc_stop_datetime)
```
Comment (mkangia) [inline corehq/motech/openmrs/handler.py]: nit: this method is getting really long and possibly a little hard to follow. Can we extract this logic out to determine start_datetime and stop_datetime in a separate method instead?

**Review by mkangia — APPROVED** [review]

File: corehq/motech/openmrs/handler.py (hunk 1 above)
Comment (kaapstorm) [reply corehq/motech/openmrs/handler.py]: Good call.
